app/js/main.js

```diff
@@ -336,6 +336,8 @@ function _removeFrameReelSelection() {
 function _addFrameReelSelection(id) {
     "use strict";
     document.querySelector(`.frame-reel-img#img-${id}`).classList.add("selected");
+    curSelectedFrame = id;
+    statusBarCurFrame.innerHTML = id;
```

Comment:
I'm not sold on these two lines. The method is called `_addFrameReelSelection`. Ideally, methods only interact with what they say they do. Now, this method says on the tin we only alter the frame reel, but it actually also edits the status bar.

Fun challenge for you: devise how this should be fixed on your own instead of me telling you. Let's see how much you are learning. :)